compiler/sem.nim

```diff
@@ -609,15 +609,19 @@ proc myProcess(context: PPassContext, n: PNode): PNode =
 
 proc testExamples(c: PContext) =
   let inp = toFullPath(c.config, c.module.info)
-  let outp = inp.changeFileExt"" & "_examples.nim"
+  let runnableExamplesDir = c.config.runnableExamplesDir
+  var outp = inp.changeFileExt"" & "_examples.nim"
+  if runnableExamplesDir.len>0:
+    discard existsOrCreateDir(runnableExamplesDir)
+    outp = runnableExamplesDir / outp.extractFilename
   let nimcache = outp.changeFileExt"" & "_nimcache"
-  renderModule(c.runnableExamples, inp, outp)
+  renderModule(c.runnableExamples, inp, outp, conf = c.config)
```

Comment:
NOTE to reviewer: Please double check this line where I added `conf = c.config`
was the intention to have nil conf?

Reply:
Your change seems to be correct.

Reply:
ok to merge then?